src/vm.ts

```diff
@@ -468,6 +474,7 @@ function checkSize(size: bigint | number, limit: number) {
 }
 
 const GATEWAY_EXT_ABI = new Interface([
+  // IReadBytesAt.sol
```

Comment:
Typo - there is no interface definition - just `ReadBytesAt.sol`

Reply:
It was `IReadBytesAt` but since the implementation is so easy, I changed it to a mixin.